tests/data/placebo/test_rds_tag_and_remove/rds.ListTagsForResource_1.json

```diff
@@ -2,21 +2,32 @@
     "status_code": 200, 
     "data": {
         "ResponseMetadata": {
+            "RetryAttempts": 0, 
```

Comment:
this one should be gone too?

Reply:
This test contains a separate call for listing the tags. I will replace that with GetResources.

Reply:
if it was the test code doing it explicitly via client in the test case thats fine to leave as is